operator/pkg/ciliumendpointbatch/endpointbatch.go

```diff
@@ -283,6 +284,12 @@ func (c *CiliumEndpointBatchController) syncCeb(key string) error {
 			return err
 		}
 	}
+	// Update metrics
+	metrics.CiliumEndpointBatchDensity.WithLabelValues("").Observe(float64(c.Manager.getCepCountInCeb(key)))
+	cepInsert, cepRemove := c.Manager.getCebMetricCountersAndClear(key)
+	metrics.CiliumEndpointsChangeCount.WithLabelValues(metrics.LabelValueCEPInsert).Observe(float64(cepInsert))
+	metrics.CiliumEndpointsChangeCount.WithLabelValues(metrics.LabelValueCEPRemove).Observe(float64(cepRemove))
+	metrics.CiliumEndpointsChangeCount.WithLabelValues(metrics.LabelValueCEPTotal).Observe(float64(cepInsert + cepRemove))
```

Comment:
Is this total necessary? A histogram should provide a "_sum" and a "_count".

From prom docs: 

> Note that the number of observations (showing up in Prometheus as a time series with a _count suffix) is inherently a counter (as described above, it only goes up). The sum of observations (showing up as a time series with a _sum suffix) behaves like a counter, too, as long as there are no negative observations.